packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -806,6 +806,17 @@ export class ChatController {
             type: 'chat_message',
             context: undefined,
         })
+
+        const toolUseWithError: ToolUseWithError = {
+            toolUse: {
+                name: ToolType.ExecuteBash,
+                input: message.action.text,
+                toolUseId: message.action.formItemValues?.toolUseId,
+            },
```

Comment:
let's first fetch it from the session and compare the name to make sure is executeBash, if it is, we just add an error to store it back to the session

Reply:
didn't get the purpose of getting session from sessionStorage. I mean for the reject, we need to set the error, right? no matter what exists in the sessionStorage.

Reply:
discussed offline that the session storage always set the toolUseWithError(without error), only need to add the error to toolUseWithError when shell command rejects.